frontend/src/components/IngredientsList.js

```diff
@@ -1,11 +1,19 @@
 import styled from "styled-components/macro";
 import IngredientItem from "./IngredientItem";
 
-export default function IngredientsList({ ingredientsList, index }) {
+export default function IngredientsList({
+  ingredientsList,
+  setIngredientsList,
+}) {
   return (
     <Wrapper>
       {ingredientsList.map((ingredient, index) => (
-        <IngredientItem key={index} ingredient={ingredient} />
+        <IngredientItem
+          key={index}
```

Comment:
Why not use the ingredient as a key? https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Because the ingredients are saved in the Database just with _id: "Name of the ingredient"

Reply:
So name of the ingredient should be unique and you can do `key={ingredient}`?

Reply:
Changed the key to ingredient, but in the wrong branch, but it will be Updated. Thanks for your advice.